tron/kubernetes.py

```diff
@@ -279,12 +279,15 @@ def __init__(
         enabled: bool = True,
         default_volumes: Optional[List[ConfigVolume]] = None,
         pod_launch_timeout: Optional[int] = None,
+        start_schedule_jobs: bool = True,
     ):
         # general k8s config
         self.kubeconfig_path = kubeconfig_path
         self.enabled = enabled
         self.default_volumes: Optional[List[ConfigVolume]] = default_volumes or []
         self.pod_launch_timeout = pod_launch_timeout or DEFAULT_POD_LAUNCH_TIMEOUT_S
+        # This flag will be used to indicate in logs the first task we start scheduling and running on Kubernetes
+        self.start_schedule_jobs = start_schedule_jobs
```

Comment:
i think we can probably skip this logic and say that we can start scheduling jobs after `self.connect()` has been called further below :)

Reply:
hmm i wonder if we should add that log message after self.connect to indicate that we connected to task_proc and to also keep the current log produced after the flag "start_schedule_jobs". Because we do all the connection stuff under ``self._run_mcp(boot_time=boot_time)`` meanwhile we actually start submitting tasks to k8s when we are waiting after we run the reactor, at this point in the code
```
            while True:
                signum = signal.sigwait(list(signal_map.keys()))
```

Reply:
the log message from the `start_schedule_jobs` flag will probably be somewhat unreliable if we succeed in lowering the restart time as it will depend on job schedules 

that said: the task_proc stuff does not run on the twisted reactor, so it should be possible for it to run beforehand

(although, I guess that until the reactor is running there won't be any new tasks submitted to task_proc)

Reply:
in general, i'm a little hesitant to add a state-indicating boolean just for logging purposes, so perhaps we can try to use the tron/trondaemon.py logging as a proxy timing metric instead?

Reply:
Actually, they both happen after the reactor runs under ``submit_command`` function. I wonder if we should still keep both though cause there are some function calls between them (i.e time spent in other functions)